### PR TITLE
Add script that writes powerblade data to file

### PR DESCRIPTION
**Pull request by rliou92**
The base of the script is written by Anand that writes the data to an
influx database.

### CODE REVIEW

File: data_collection/advertisements/powerblade_adv_influx.js [hunk 1]
```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env node
+// original file: https://github.com/lab11/powerblade/tree/master/data_collection/advertisements
+
+var noble = require('noble');
+var config = require('./config');
+var influx;
+
+function getDateTime() {
+
+    var date = new Date();
```
Comment (brghena) [inline data_collection/advertisements/powerblade_adv_influx.js:10]: This probably works fine, so no point changing it now, but in the future you might want to use libraries such as [Moment](http://momentjs.com/docs/#/displaying/) to do this kind of thing with a format string.

File: data_collection/advertisements/powerblade_adv_influx.js [hunk 2]
```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env node
+// original file: https://github.com/lab11/powerblade/tree/master/data_collection/advertisements
+
+var noble = require('noble');
+var config = require('./config');
```
Comment (brghena) [inline data_collection/advertisements/powerblade_adv_influx.js:5]: Can you add a description of what this config file should look like to the README.md file in this folder? That will allow other people in the future to use this script.

File: data_collection/advertisements/powerblade_adv_influx.js [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+var UMICH_COMPANY_ID = 0x02E0;
+var POWERBLADE_SERVICE_ID = 0x11;
+var OLD_COMPANY_ID = 0x4908;
+
+// {BLE Address: Most recent sequence number} for each PowerBlade
+var powerblade_sequences = {};
+
+var encoding = 'utf8';
+const fs = require('fs');
+var stream = fs.createWriteStream("powerblade_data.txt", {encoding:encoding,flags:'w'});
```
Comment (brghena) [inline data_collection/advertisements/powerblade_adv_influx.js:80]: You should add to the README instructions on how to run this script and note that it creates (or overwrites if existing) the file `powerblade_data.txt` with results.

**Review by brghena — CHANGES_REQUESTED** [review]
This is looks pretty good. You should update the README with a little bit of documentation like I noted, but after that, this is good to go.

File: data_collection/advertisements/powerblade_adv_influx.js (hunk 1 above)
Comment (rliou92) [reply data_collection/advertisements/powerblade_adv_influx.js:10]: Thanks, I did not know about that library.

File: data_collection/advertisements/powerblade_adv_influx.js (hunk 2 above)
Comment (rliou92) [reply data_collection/advertisements/powerblade_adv_influx.js:5]: Added

**Review by brghena — APPROVED** [review]

**Comment by brghena** [comment]
Looks good. Thanks!

**Pull request merged**